docs/gen-ai/gen-ai-spans.md

```diff
@@ -109,45 +105,173 @@ If none of these options apply, the `gen_ai.system` SHOULD be set to `_OTHER`.
 
 ## Events
 
-In the lifetime of a GenAI span, an event for prompts sent and completions received MAY be created, depending on the configuration of the instrumentation.
+GenAI instrumentation MAY capture user inputs send to the model and responses received from it.
+
+It's RECOMMENDED to use [Event API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md) to record GenAI events once it's implemented in corresponding language (check out [Spec Compliance Matrix](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md#events) for the details).
 
-<!-- semconv gen_ai.content.prompt -->
+The event payload describes message content sent to or received from GenAI and depends on specific messages described in the following sections.
+
+Instrumentations for individual GenAI systems MAY add system specific fields into corresponding events payload.
+It's RECOMMENDED to document them in system-specific extensions.
+Telemetry consumers SHOULD expect to receive unknown payload fields.
+
+### Configuration
+
+Instrumentations that support capturing inputs and outputs, SHOULD NOT capture them by default and SHOULD offer the ability to turn on capturing of prompts and completions. This is for three primary reasons:
+
+1. Data privacy concerns. End users of GenAI applications may input sensitive information or personally identifiable information (PII) that they do not wish to be sent to a telemetry backend.
+2. Data size concerns. Although there is no specified limit to sizes, there are practical limitations in programming languages and telemetry systems. Some GenAI systems allow for extremely large context windows that end users may take full advantage of.
+3. Performance concerns. Sending large amounts of data to a telemetry backend may cause performance issues for the application.
+
+Instrumentations MAY offer additional configuration options allowing to control sensitive content is capture.
```

Comment:
change to "SHOULD offer", default to events with content redacted - we can change later based on the feedback